library/spit_results.py

```diff
@@ -39,10 +199,17 @@ def write_to_csv(self):
             file_size = os.path.getsize(f_path)
             vals = self.vals
             fields = sorted(vals[0].keys())
+            try:
+                fields.remove('systemid.contents')
```

Comment:
I've pushed a commit to separate these two removals. I've tried to only do minimal changes here base on the direction you mentioned on Friday. Looking forward to the PR.